orchagent/vnetorch.h

```diff
@@ -326,6 +357,11 @@ class VNetRouteOrch : public Orch2, public Subject
     VNetNextHopObserverTable next_hop_observers_;
     std::map<std::string, VNetNextHopGroupInfoTable> syncd_nexthop_groups_;
     std::map<std::string, VNetTunnelRouteTable> syncd_tunnel_routes_;
+    BfdSessionTable bfd_sessions_;
+    std::map<std::string, VNetEndpointInfoTable> nexthop_info_;
+    ProducerStateTable m_bfdSessionProducer;
```

Comment:
could you please use the same naming convention as in the rest of class

Reply:
Updated naming to match naming convention.